scipy/stats/_continuous_distns.py

```diff
@@ -3276,6 +3271,7 @@ def _entropy(self):
         return _EULER + 1.
 
     @_call_super_mom
+    @inherit_docstring_from(rv_continuous)
```

Comment:
`fit` method needs to inherit the superclass docstring.
Not sure why `functools.wraps` in `_call_super_mom` is not taking care of this.